Final project proposal/Proposal_Yixuan&Hang4.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Applying AI to identify spatial clustering of pedestrian activity and street environment features"
+author: Yixuan Zhao & Hang Tian
+output: 
+  github_document
+---
+
+# Introduction to problem/question
+
+Encouraging walking activities is gaining increasing attention as a significant strategy for improving public health and urban revitalization.
+
+There are plenty of research focusing on impact of the built environment on physical activities; However, few have studied the mixed spatial relationship and variation between pedestrian activities and urban security environment, community quality of life and built environment characteristics. The potential relationship between spatial patterns of pedestrian activities, aggregation or dispersion, and environmental features is critical, which will not only direct urban planners to focus pedestrian safety issues, but also lead architects to consider providing a comfortable walking experience around the city.
+
+Under the topic of pedestrian activities and their dynamics, questions addressing issues such as overcrowding and vehicle-pedestrian collision mitigation have been thoroughly looked into while pedestrian activities itself is rarely a focal point. Nowadays, with the advancement of remote scanning technology, it has been used in many research topics including intelligent transportation systems and public safety evacuation. Therefore, monitoring pedestrian activities is becoming easier for understanding complex collective behavior in social systems.
+
+# Problem / Question
+
+Our research hypothesis: (1) Violent crime has a negative impact on the number of street pedestrians, while property crime is positively correlated with the number of street pedestrians; (2) Different types of pedestrian generators will have different impacts on pedestrians. We believe that high-density catering and retail pedestrian generators and perfect infrastructure will bring more pedestrians to the area; (3) Neighborhoods with low social satisfaction and streets near disadvantaged communities will not attract too many pedestrians.

```

Comment:
love the clear hypotheses - though think through other factors that could explain patterns that you are not including (income, etc.).  Also be careful inferring cause and effect relationships from correlations.